addons/purchase/wizard/purchase_order_cancel.xml

```diff
@@ -0,0 +1,21 @@
+<odoo>
+    <record id="view_purchase_cancel_wizard" model="ir.ui.view">
+        <field name="name">purchase.cancel.wizard.form</field>
+        <field name="model">purchase.cancel.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Cancel Purchase Orders">
+                <field name="purchase_order_ids" invisible="1"/>
+                <field name="purchase_count" invisible="1"/>
```

Comment:
Since this PR: https://github.com/odoo/odoo/pull/137031, we no longer have to put invisible fields in the view to use them :tada:, so I think we can drop these two lines